src/Symfony/Bundle/SecurityBundle/DataCollector/SecurityDataCollector.php

```diff
@@ -136,16 +137,31 @@ public function collect(Request $request, Response $response, \Exception $except
 
         // collect voters and access decision manager information
         if ($this->accessDecisionManager instanceof TraceableAccessDecisionManager) {
-            $this->data['access_decision_log'] = $this->accessDecisionManager->getDecisionLog();
-            $this->data['voter_strategy'] = $this->accessDecisionManager->getStrategy();
-
-            foreach ($this->accessDecisionManager->getVoters() as $voter) {
-                $this->data['voters'][] = $this->hasVarDumper ? new ClassStub(get_class($voter)) : get_class($voter);
+            $decisionLog = $this->accessDecisionManager->getDecisionLog();
+
+            // Voter constants
+            $reflexionClass = new \ReflectionClass(VoterInterface::class);
+            // Allows to get the access constant name from the vote log
+            $voterConstants = array_flip($reflexionClass->getConstants());
```

Comment:
I am not convinced that we really need to use reflection here. Doing some hardcoded checks below is probably good as well.

Reply:
@xabbuh Modified with hardcoded values, thanks